op-batcher/batcher/channel_config_provider.go

```diff
@@ -50,7 +50,14 @@ func NewDynamicEthChannelConfig(lgr log.Logger,
 // calldata and for blobs, given current market conditions: it will return
 // the appropriate ChannelConfig depending on which is cheaper. It makes
 // assumptions about the typical makeup of channel data.
-func (dec *DynamicEthChannelConfig) ChannelConfig(isPectra bool) ChannelConfig {
+//
+// The blob config is returned when throttling is in progress.
```

Comment:
Perhaps expand this comment a bit with the justification? i.e. we want to prioritize throughput even if that means paying more.